install/kubernetes/microcks/values.yaml

```diff
@@ -23,10 +23,10 @@ microcks:
   resources:
     requests:
       cpu: 200m
-      #memory: 400Mi
+      memory: 400Mi
     limits:
       #cpu: 500m
-      memory: 512Mi
+      memory: 400Mi
```

Comment:
I haven't run new benchmarks recently (with Java 17 and new SpringBoot 3.1) and I remember `512Mi` was previously chosen as a compromise between resource consumption and startup time (according to probes settings as well). Have you performed new measurements that could make us change to `400Mi`?

Reply:
I haven't run any benchmark and I based the new request/limit value on the the one that was commented out: 400Mi.

> I haven't run new benchmarks recently (with Java 17 and new SpringBoot 3.1) and I remember 512Mi was previously chosen

I will modify the PR accordingly to your benchmark